src/vis/vis.js

```diff
@@ -1208,7 +1208,16 @@ var Vis = View.extend({
 
     var mapView = this.mapView;
     var infowindow = null;
-    var layers = layerView.model && layerView.model.layers || [];
+    var layers = [];
+    if (layerView.model) {
+      if (layerView.model.layers) {
+        layers = layerView.model.layers;
+      } else {
+        if (layerView.model.getInfowindowData) {
+          layers = new Backbone.Collection([layerView.model]);
```

Comment:
not sure if this is the best way, should we do add the interactive layers have "sublayers" ? cc @alonsogarciapablo